src/api/chainApi/types.ts

```diff
@@ -76,6 +76,9 @@ export interface IMarketController {
   transferBidBalance: (from: string, amount: string, options: TransactionOptions) => Promise<void>
   transferBalance: (from: string, to: string, amount: string, options: TransactionOptions) => Promise<void>
   getKusamaFee: (sender: AddressOrPair, recipient?: string, value?: BN) => Promise<Balance>
+  setCollectionSponsor(collectionId: number, sponsorAddress: string, options: TransactionOptions): Promise<void>
```

Comment:
those are methods of collections (rpc), not market (contract). Better to stick them inside collection controller instead.